packages/trpc/server/routers/viewer/teams/createInvite.handler.ts

```diff
@@ -41,6 +41,7 @@ export const createInviteHandler = async ({ ctx, input }: CreateInviteOptions) =
       identifier: `invite-link-for-teamId-${teamId}`,
       token,
       expires: new Date(new Date().setHours(168)), // +1 week,
+      expiresInDays: 7,
```

Comment:
I don't this is required. check the above link.

Reply:
expires is already set to one week later

Reply:
Yeah but when if we don't update it, expiresInDays is set to null

https://github.com/Amit91848/cal.com/blob/9cdb942ad8ae6b3ce3be49dc8243f64b0bb94253/packages/features/ee/teams/components/InviteLinkSettingsModal.tsx#L86-L91

Here defaultValue will be selected as never expire

Reply:
https://github.com/calcom/cal.com/assets/74371312/1452941c-c37c-40da-8065-0db7de638959

Reply:
https://github.com/calcom/cal.com/assets/53316345/d3cfafad-1ebf-41a0-b333-d1887aaa55bf

Still don't see default as 7 days

Reply:
we can do this later

Reply:
I guess you are copying already existing link with never expire set, try deleting and trying again